From: Marc Kleine-Budde <m.kleine-budde@pengutronix.de>
Date: Wed, 25 Jan 2012 17:50:07 +0100
Subject: [PATCH] pppd: make makefile sysroot aware

The pppd Makefile is not SYSROOT aware. This patch replaces all
occurrences of absolute paths by their corresponding SYSROOT relative
ones.
Skip the 'wildcard' check completely. Libcrypt is always needed anyways.
Drop search paths in /usr/local and /usr/kerberos. Those are not needed and
don't exist anyways.

Signed-off-by: Marc Kleine-Budde <m.kleine-budde@pengutronix.de>
Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 pppd/Makefile.linux | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pppd/Makefile.linux b/pppd/Makefile.linux
index bc01e3fd2a24..9b0119463c1f 100644
--- a/pppd/Makefile.linux
+++ b/pppd/Makefile.linux
@@ -111,8 +111,8 @@ endif
 
 # EAP SRP-SHA1
 ifdef USE_SRP
-CFLAGS	+= -DUSE_SRP -DOPENSSL -I/usr/local/ssl/include
-LIBS	+= -lsrp -L/usr/local/ssl/lib
+CFLAGS	+= -DUSE_SRP -DOPENSSL
+LIBS	+= -lsrp
 TARGETS	+= srp-entry
 EXTRAINSTALL = $(INSTALL) -c -m 555 srp-entry $(BINDIR)/srp-entry
 MANPAGES += srp-entry.8
@@ -146,7 +146,7 @@ endif
 
 ifdef NEEDDES
 ifndef USE_CRYPT
-CFLAGS   += -I$(shell $(CC) --print-sysroot)/usr/include/openssl
+CFLAGS   += -I$(SYSROOT)/usr/include/openssl
 else
 CFLAGS   += -DUSE_CRYPT=1
 endif
